### ISSUE TITLE
Clock simply disappeared at some time

### ISSUE THREAD

**Issue by hesspoint** [ngnijland/MMM-text-clock#35]
Hi,

I had been using the module without issues for a long time. Then all of a sudden it disappeared. There is simply black screen where it used to be. I did not change anything in the config. With the latest update of MM I had issues so I did a complete reinstall of MM and hoped that would also fix it but it didn't.

I am now on the latest MM from a few days ago. Also latest version of the module.

This is my config

            {
		    module: "MMM-text-clock",
		    position: "top_left",
		    config: {
		    	size: "small"
		    }
		},

No idea why it is not working anymore

**Comment by ngnijland** [comment]
Hi @hesspoint!

I'm trying to reproduce the crash, but had no luck yet (I let the latest version of the module run on its own for an entire night while using the latest version of MM). 

To debug this further there is a few things you can do:
1. Try to run MM with only the `MMM-text-clock` configured, to double check if it's a bug in this module
2. The next time the clock disappears could you check both the Chromium console and the node server for error logs?

You can post the logs here if you want. Then I'll give them a quick look.

Let me know if you have any questions!

**Comment by sainiaj** [comment]
yes not displaying for me either for now, I am using the default clock module, but I loved this text clock tbh

**Comment by ngnijland** [comment]
Hi @sainiaj !

Could you share your config and/or any error logs that show up when this module doesn't work as expected?

**Comment by sainiaj** [comment]
Can you please show me, how to see the log file
Thank you

**Comment by sainiaj** [comment]
@ngnijland 

		 {
    module: "MMM-text-clock",
    position: 'top_left',
    config: {
      // Options
    }
  },

config.js 

I am not sure where to check the error log can you please point me to right direction

Thank you

**Comment by ngnijland** [comment]
@sainiaj that config file looks fine to me. 

There is two places were you can check logs:
1. The terminal window were you run `npm run start` to start MagicMirror
2. The chrome console. MagicMirror runs in a fullscreen browser. Right click anywhere on the page and click "Inspect"

Let me know if you find anything that looks out of the ordinary.
